schemas/+lab/Location.m

```diff
@@ -4,6 +4,8 @@
 location:                   varchar(32)
 -----
 location_description='':    varchar(255)
+acquisition_type='':        varchar(128)     #to define if it's mesoscope or 2_3photon imaging
```

Comment:
to make it searchable, you may want to create another table called LocationType and put in entries such as behavior only, meso, 2 photon, 3 photon, and make this current field point to the table. If you don't quite need to query on this field, then your design here is fine.

Reply:
Yes, I think the way you are describing is cleaner.
I will change it to be as you describe.
I think the name of the table should be acquisition_type.